### DISCUSSION TITLE
cli command: app -> use

### DISCUSSION THREAD

**Discussion by buschNT** [unikubehq/cli#60] [General]
Not implemented: `unikube app use`

**Comment by buschNT** [comment]
I am not sure if we need this command (at least currently)?
What do you think @Schille ?

Do we really want to set an app, such as "buzzword-counter-web-574569d7dc-56j5x", into the context?

**Reply by Schille** [reply]
I am unsure. I think we have an unclear definition of _app_ in my opinion. Why don't we move the entire `app` command group to deck and use `deck use` instead. Does it make more sense? - what is the difference between _app_ and _deck_ anyway?
@SteinRobert

**Reply by SteinRobert** [reply]
Not sure if that even makes sense, from my understanding it does not. What's the use case for setting an app into context?

**Reply by schwobaseggl** [reply]
Aren't there multiple apps per deck? Each of which can be "shelled" or has its own env, so `app shell` and `app env` make sense to me. As a user, I would expect commands from the deck group to take a deck as argument which seems illogical for the two mentioned.
I would on the other hand suggest to move the ingress command to the deck level, as this is where the ingresses are defined and not every app in a deck must be exposed outside the cluster.

**Reply by Schille** [reply]
That's relatable. We can go that path but anyway I'd like to have a better understanding of an app. A deck may contain one or many apps. What's an app in the context of a docker-compose file? The entire mesh or just a particular service? Maybe we should rename app as well? What do you think? Basically an "app" is something which is switchable ( K8s deployment, statefulset) at the moment.
